vars/cloudFoundryDeploy.groovy

```diff
@@ -412,17 +413,29 @@ private deploy(def cfApiStatement, def cfDeployStatement, def config, Closure po
         usernameVariable: 'username'
     )]) {
 
+        def postDeployCommand = config.postDeployCommand
+
+        if(postDeployCommand) {
```

Comment:
why not using `config.postDeployCommand` directly

Reply:
So what happened was I was unsure my new postDeployCommand was getting set and echoed it out for confirmation.  Had already changed the if statement so left it this way.
Yes, config.postDeployCommand would work fine.